### PR TITLE
GrandmaBettyHarveysBrownies

### PR DESCRIPTION
**Pull request by markgharvey**
Uploading GrandmaBettyHarveysBrownies

### CODE REVIEW

File: recipes/desserts/GrandmaBettyHarveysBrownies.adoc [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+A family recipe for brownies that don't dry out & get rock like.  
+
+Excellent with simple vanilla ice cream or whipped cream.
+//tell some background about it, why you like it, how it tastes ...
+
+[width="100%",cols="3,3,10,10",options="header"]
+|=========================================================
+|Metric | US        | Ingredient         | 
+
+|       | 1/2 stick | butter, room temperature |
```
Comment (dcermak) [inline recipes/desserts/GrandmaBettyHarveysBrownies.adoc:33]:
```suggestion
| 75 g | 1/2 stick | butter, room temperature |
```
unsalted butter, I assume?

File: recipes/desserts/GrandmaBettyHarveysBrownies.adoc [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+|       | 1 cup     | white sugar |
```
Comment (dcermak) [inline recipes/desserts/GrandmaBettyHarveysBrownies.adoc:34]:
```suggestion
| 200 g | 1 cup     | white sugar |
```

File: recipes/desserts/GrandmaBettyHarveysBrownies.adoc [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+|       | 2         | eggs |
+|       | 1/2 cup   | all purpose flour |
```
Comment (dcermak) [inline recipes/desserts/GrandmaBettyHarveysBrownies.adoc:36]:
```suggestion
| 60 g | 1/2 cup   | all purpose flour |
```

File: recipes/desserts/GrandmaBettyHarveysBrownies.adoc [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+|       | 1/2 cup   | chopped pecans or walnuts |
+|       | 1/2 cup + 1 tablespoon | cocoa powder |
```
Comment (dcermak) [inline recipes/desserts/GrandmaBettyHarveysBrownies.adoc:38]:
```suggestion
| 65 g | 1/2 cup + 1 tablespoon | cocoa powder |
```

File: recipes/desserts/GrandmaBettyHarveysBrownies.adoc [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+|       | 3 tablespoons | canola oil |
```
Comment (dcermak) [inline recipes/desserts/GrandmaBettyHarveysBrownies.adoc:39]:
```suggestion
| 45 ml | 3 tablespoons | canola oil |
```

File: recipes/desserts/GrandmaBettyHarveysBrownies.adoc [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+|       | 1 teaspoon    | vanilla |
```
Comment (dcermak) [inline recipes/desserts/GrandmaBettyHarveysBrownies.adoc:40]:
```suggestion
| 5 ml | 1 teaspoon    | vanilla extract |
```

**Review by dcermak — COMMENTED** [review]
I've added suggestions including metric units

**Review by rsimai — CHANGES_REQUESTED** [review]
[id='sec.name_your_dish'] could be (e.g.) 'sec.GrandmaBettyHarveysBrownies'

**Comment by bmwiedemann** [comment]
Please note that repo license changed to CC-BY-SA 4.0

**Comment by rsimai** [comment]
@jeremy-moffitt would you know if the id in the recipe is OK with spaces?

**Comment by jeremy-moffitt** [comment]
> @jeremy-moffitt would you know if the id in the recipe is OK with spaces?

It seems to work ok both in the github preview and in the build (no warnings), I'm assuming since its a quoted string that just about anything that can be url encoded can go in there. In my local copy it translates the current value to `sec.Grandma%20Betty%20Harvey` . A quick look at the asciidoctor docs indicates that its really driven by the output, so if we wanted to export to docbook (AFAIK we do not) there would be problems with ids that start with numbers for example. As long as we only care about the asciidoc rendering and the html rendering, we should be ok with spaces.

**Comment by rsimai** [comment]
Thank you, let's merge then.

**Review by rsimai — APPROVED** [review]

**Pull request merged**